text/0000-project-groups.md

```diff
@@ -0,0 +1,328 @@
+    difficulties encountered, and what they would want to be improved.
+
+8. Archive infrastructure.
+
+- Archive GitHub repository to be read-only.
+- Archive chat channel(s) on any platforms.
+
+9. (Optional) Restore group
+
+- At any later point the group could be restored to active status if there's
+  interest among the team to revive the effort.
+- If significant time has passed, part of restoring the group should be to
+  evaluate whether the past decisions and rationale are still applicable to the
+  present.
+- Go to step 2.
+
+[`rust-lang/team`]: https://github.com/rust-lang/team
+[internals]: https://internals.rust-lang.org
+
+# Reference-level explanation
+
+A Project Group is a group of people working on a particular project or
+responsibilities at the behest of an official Rust team. Some project groups are
+are ephemeral, meaning that they are archived once the project is complete.
+However, there are project groups that have continual work and maintenance.
+
+Examples of project groups around specific feature  include [FFI Unwind],
+[Inline ASM], and [Safe Transmute]. Examples built around continual work and
+maintenance would include [Triage], and [Rustup].
+
+The goal of a project is build a community or formalise and existing community
+around a particular feature or project in the in the organisation, and use this
+space to discuss and iterate on that feature.
+
+Part of building a community is removing some of the institutional memory that
+develops in the design process, and centralising the information and discussion
+around the feature so that we can provide better visibility into why certain
+decisions and trade offs were made over others.
+
+Previously a lot of the discussion and iteration for large features would
+happen in the initial RFC thread. This leads to a lot of discussion in the top
+of the thread and that often becomes completely irrelevant to the current
+iteration.
+
+This process has also been unsuitable to describe features that can take
+multiple years to develop and will become multiple RFCs over the course of its
+design process. Some examples of of this are the "`impl Trait`" and "macros
+2.0" features, where the goals has shifted a lot from the initial RFCs, and it
+can be hard to know their current status.
+
+[ffi unwind]: https://github.com/rust-lang/project-ffi-unwind
+[inline asm]: https://github.com/rust-lang/project-inline-asm
+[safe transmute]: https://github.com/rust-lang/project-safe-transmute
+[triage]: https://github.com/rust-lang/team/blob/master/teams/wg-triage.toml
+[rustup]: https://github.com/rust-lang/team/blob/master/teams/wg-rustup.toml
+
+## Creation
+
+Since project groups are approved by their relevant parent team over the core
+team, it's up to each team decide their specific requirements. However the author
+recommends that a group should try to address at least of the following questions.
+
+- What value do you see your group bringing to the organisation?
```

Comment:
I think there's also another question here that's worth adding: Why should this be a project group vs. a community effort? i.e. what are you expecting from the Rust org?

(I see that this is somewhat asked below in "What do you expect the relationship to the organisation be?" but I would want to be more explicit and have that as one of these three points).